hack/tools.sh

```diff
@@ -123,6 +124,42 @@ install_yq() {
 	ok "yq was installed successfully"
 }
 
+go_install() {
+	local pkg="$1"
+	local version="$2"
+	shift 2
+
+	info "installing $pkg version: $version"
+
+	GOBIN=$LOCAL_BIN \
+		go install "$pkg@$version" || {
+		fail "failed to install $pkg - $version"
+		return 1
+	}
+	ok "$pkg - $version was installed successfully"
+
+}
+
+validate_verison() {
```

Comment:
Love the `validate_version` addition. Just a thought since we have it here maybe we can update other tools to make use of it as well?

Reply:
yes, I thought of it but I felt it is a feature creep. And it can be done is a different PR.